opentracing/tracer.py

```diff
@@ -82,3 +83,24 @@ def close(self):
         fut = Future()
         fut.set_result(True)
         return fut
+
+    def implementation_id(self):
+        """Returns the ImplementationID for this OpenTracing implementation.
+
+        :return: An ImplementationID instance that describes this OpenTracing
+            implementation.
+        """
+        return ImplementationID('noop', '1.0.0')
+
+
+# Per collections.namedtuple, the below defines a class called ImplementationID
+# which can be initialized via positional parameters or a kwargs-style
+# initializer.
+#
+#     dapper_impl = ImplementationID('dapper', '0.1.2')
+#     zipkin_impl = ImplementationID(name='zipkin', version='0.3.4')
+#
+ImplementationID = namedtuple('ImplementationID', 'name version')
+
+# The SemVer (http://semver.org/) of the OpenTracing Python API.
+OPENTRACING_PYTHON_SEMVER = '0.9.0'
```

Comment:
@bcronin I see what you're saying. If we're going through the trouble of including this API in the first place, we may as well do it right. I'm not at my laptop right now, but I will think about it... It's annoying that Tracer impls need not actually inherit from Tracer (due to the usual dynamic typing situation)... If that weren't the case, this would be straightforward! Maybe we should just require it in a clearly worded comment or something...